### PR TITLE
(MAINT) Fix errant docstring

### PR DESCRIPTION
**Pull request by cprice404**

### CODE REVIEW

**Comment by cprice404** [comment]
@nwolfe the thing you caught in your review of the merge-up PR

File: ruby/puppet [hunk 1]
```diff
@@ -1 +1 @@
-Subproject commit 8e2e03ee7a639d4c2c0caad0b91c3727c479f319
+Subproject commit 4dae4b348ab6942d07c178fd160726e2c7895eaa
```
Comment (camlow325) [inline ruby/puppet]: Probably didn't mean to include this?

**Comment by camlow325** [comment]
:+1: otherwise

File: ruby/puppet (hunk 1 above)
Comment (cprice404) [inline ruby/puppet]: submodules--

**Comment by cprice404** [comment]
@camlow325 fixed

**Pull request merged**